api_server.py

```
@@ -549,4 +557,10 @@ def add_security_headers(response):
 """
 
 if __name__ == "__main__":
+    if os.environ.get("FLASK_ENV") == "production":
+        logger.error(
+            "Security Risk: Flask development server started in production. "
+            "Use a production WSGI server (e.g., gunicorn) instead."
+        )
+        sys.exit(1)
     app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))
```

Ruff (0.14.14)

[error] Possible binding to all interfaces

(S104)